src/client/src/components/pages/managers/addManager/index.jsx

```diff
@@ -24,17 +26,55 @@ export default class index extends Component {
     this.setState(fields);
   };
 
+  addManager = async obj => {
+    await swal({
```

Comment:
Sorry for the confusing advice - I think this alert *does* need an if-statement afterwards, i.e. `const confirm = await swal() .... if (confirm.value) {`, since here, the user could click the thumbs-down and then it should not add the manager.

Reply:
Yes I got it now :smile: 
I edited it now and I will do it in the other branch

Reply:
Cool! Thanks @mohannadhanafi!